src/views/StoredTags.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <!-- Manage Tags -->
+  <div>
+    <!-- Top header as a level -->
+    <nav class="level">
+      <!-- Left side -->
+      <div class="level-left">
+        <div class="level-item">
+          <p class="title">
+            Manage Tags
+          </p>
+        </div>
+      </div>
+
+      <!-- Right side -->
+      <div class="level-right">
+        <div class="level-item">
+          <div class="field has-addons">
+            <p class="control">
+              <input class="input" type="text" placeholder="Add a Tag" v-model="newTag.title" />
+            </p>
+            <p class="control">
+              <button class="button is-info" @click="createTags">
+                Add
+              </button>
+            </p>
+          </div>
+        </div>
+        <div class="level-item">
+          <p class="has-text-danger">
+            Tag cannot be blank and must be one word and alphanumeric.
+          </p>
+        </div>
+      </div>
+    </nav>
+
+    <!-- Table to show tags -->
+    <div class="section" style="height:60vh;overflow-x:auto;overflow:auto;">
+      <table class="table is-striped is-fullwidth">
+        <thead>
+          <tr>
+            <th>Tag</th>
+            <th></th>
+            <th>Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+          <thead>
+            <tr>
+              <th>Active Tags</th>
+            </tr>
+          </thead>
+          <tag-entry
+            v-for="tag in tagsList.results"
+            v-bind:key="tag.id"
+            v-bind:tagId="tag.id"
+            v-bind:teamId="teamId"
+            v-bind:tagTitle="tag.title"
+            v-on:update="getTags"
+          >
+          </tag-entry>
+        </tbody>
+      </table>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, ref, onMounted } from "vue";
+import { TeamRecord, getTeam } from "@/api/teams";
+import { PaginatedList } from "@/api/base";
+import TagEntry from "./TagEntry.vue";
+import {
+  createTag,
+  listTag,
+  TagRecord,
+  WriteTagRecord,
+} from "@/api/tags";
+import store from "@/store";
+export default defineComponent({
+  components: { TagEntry },
+  name: "Tags",
+  props: {
+    teamId: {
+      type: String,
+      required: true,
+    },
+  },
+  setup(props) {
+    const teamRecord = ref({} as TeamRecord);
+    const tagsList = ref({} as PaginatedList<TagRecord>);
+    const listPage = ref(1);
+    const newTag = ref({} as WriteTagRecord);
+
+    const getTeamRecord = async () => {
+      const axiosInstance = store.getters.generateAxiosInstance;
+      const teamId = parseInt(props.teamId);
+      const team = await getTeam(axiosInstance, teamId);
+      if (team) {
+        teamRecord.value = team;
```

Comment:
getTeam will either return a team record or throw and exception, so this check may not be necessary